src/DeVinci_frontend/components/Sidebar.svelte

```diff
@@ -38,7 +39,7 @@ import { onMount } from 'svelte';
   });
 
   // Reactive statement to check if the user has already downloaded at least one AI model
-  $: userHasDownloadedAtLeastOneModel = userHasDownloadedModel();
+  $: userHasDownloadedAtLeastOneModel = $downloadedModels.length > 0;
```

Comment:
best to use the existing function userHasDownloadedModel as it implements this already